tests/test_read_configs_include.py

```diff
@@ -83,17 +83,21 @@ class TestReadConfigs:
     ]
 
     def run_func(
-            self, include_file: str, local_env: Union[str, dict], command_suffix: str = ""
+            self, include_file: str, local_env: Union[str, dict], command_suffix: str = "", exit_on_error: bool = True
     ) -> subprocess.CompletedProcess:
         try:
             with tempfile.NamedTemporaryFile(delete=False, mode="w") as f:
                 set_local_env(f, local_env)
 
-            env = {"BIRDHOUSE_LOCAL_ENV": f.name}
+            env = {**os.environ, "BIRDHOUSE_LOCAL_ENV": f.name}
```

Comment:
Looks like this might lead to different test results depending on whatever is defined in anyone's personal env. Can't there be explicit variables to replicate desired test cases? Not sure what requires this addition to make the test "work" now.

Reply:
Yes, but we were always going to get different environments based on people's environments/architectures, etc.

This is specifically to set terminal related variables like `$TERM` which will raise an error if not set (and if `set -e` is set).

I could try to see if just setting `$TERM` or a smaller subset of the environment works as well.

Reply:
The point of the tests is to have a reliable and replicable way to test a predefined set of configuration. 

If we need to evaluate different combinations of environments or known problematic configurations, `@pytest.mark.parameterize` could be used to test those combinations explicitly.